app/controllers/ResourceLinksController.scala

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package controllers
+
+import javax.inject.Inject
+import play.api.libs.json.Json
+import play.api.mvc.Action
+import play.api.mvc.AnyContent
+import play.api.mvc.ControllerComponents
+import services.ListRetrievalService
+import uk.gov.hmrc.play.bootstrap.backend.controller.BackendController
+
+import scala.concurrent.ExecutionContext
+
+class ResourceLinksController @Inject() (
+  cc: ControllerComponents,
+  listRetrievalService: ListRetrievalService
+)(implicit ec: ExecutionContext)
+    extends BackendController(cc) {
+
+  def resourceLinks: Action[AnyContent] =
+    Action.async {
+      implicit request =>
+        listRetrievalService.getResourceLinks().map {
+          case Some(resourceLinks) => Ok(Json.toJsObject(resourceLinks))
+          case None                => InternalServerError("No resource links found")
```

Comment:
Should this be an InternalServerError. If no links are available should it not be a 404?

Reply:
Hey @chris-ianson, this is a good point, I was referencing the architecture document when I made this decision (https://confluence.tools.tax.service.gov.uk/pages/viewpage.action?pageId=203392719). Happy to discuss adding a 404 if we feel that adding another error code is required (Amaal has already suggested better logging here which may also be a solutions)

Reply:
Happy to leave it as is. It should always bring back some resources after initial seeding of data so if it doesn't I guess we could say somethings broken